models/sap__0fi_gl_14.sql

```diff
@@ -207,11 +207,11 @@ final as (
         and bkpf.belnr = bseg.belnr
         and bkpf.gjahr = bseg.gjahr
         and faglflexa.buzei = bseg.buzei
-    {% if var('bkpf_mandt_var',[]) and var('faglflexa_rldnr_var',[]) %}
-    where bkpf.mandt = '{{ var('bkpf_mandt_var') }}'  
-    and faglflexa.rldnr = '{{ var('faglflexa_rldnr_var') }}'
-    {% endif %}
+    {% if var('bkpf_mandt_var',[]) and var('faglflexa_rldnr_var',[]) %} 
+    where bkpf.mandt = '{{ var('bkpf_mandt_var') }}'   
+    and faglflexa.rldnr = '{{ var('faglflexa_rldnr_var') }}' 
+    {% endif %} 
```

Comment:
Same comment from a similar situation in this PR. This conditional will only run if all the variables are populated. Is that expected? If so we should call it out. If not, we should update the logic to account for situations where only one variable is provided.

Reply:
Also be sure to add double quotes instead of single